app/components/hello-react.jsx

```diff
@@ -1,8 +0,0 @@
-import '../scss/hello-react';
```

Comment:
So you removed hello-react component, but test is still there and failing, not good ;)